src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/IServiceProviderIsService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Optional service used to determine if the specified type is resolvable by the <see cref="IServiceProvider"/>.
```

Comment:
```suggestion
    /// Optional service used to determine if the specified type is available from the <see cref="IServiceProvider"/>.
```

It may be available, but it's not necessarily resolvable.

Reply:
(Same change below.)

Reply:
Done